<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diploma di riconoscimento</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/diploma.png</normaloff>:/images/diploma.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="scuola">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Inserisci scuola</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/scuola.png</normaloff>:/images/scuola.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="alunni">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Inserisci alunni</string>
         </property>
         <property name="icon">
          <iconset resource="../images.qrc">
           <normaloff>:/images/system-users.png</normaloff>:/images/system-users.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionEsci"/>
   </widget>
   <widget class="QMenu" name="menuInformazioni">
    <property name="title">
     <string>I&amp;nformazioni</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuVisualizza">
    <property name="title">
     <string>&amp;Visualizza</string>
    </property>
    <addaction name="actionAnno_scolastico"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVisualizza"/>
   <addaction name="menuInformazioni"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionEsci">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Esci</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/svn_status.png</normaloff>:/images/svn_status.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAnno_scolastico">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/calendario.png</normaloff>:/images/calendario.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Anno scolastico</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>scuola</tabstop>
  <tabstop>alunni</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
